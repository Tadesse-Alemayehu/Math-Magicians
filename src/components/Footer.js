import React from 'react';

export default function Footer() {
  return (
    <div id="footer-container">
      <h2>
        Math Magician project
      </h2>
      <h3>By, Tadesse Alemayehu.</h3>
    </div>
  );
}
