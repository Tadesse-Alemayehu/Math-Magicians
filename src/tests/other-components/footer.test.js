import renderer from 'react-test-renderer';
import Footer from '../../components/Footer';
// import useLocation from 'react'
it('Make sure the Navbar renders the same', () => {
  const navbarSnap = renderer
    .create(<Footer />)
    .toJSON();
  expect(navbarSnap).toMatchSnapshot();
});
