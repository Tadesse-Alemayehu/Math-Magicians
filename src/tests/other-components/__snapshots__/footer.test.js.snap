// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make sure the Navbar renders the same 1`] = `
<div
  id="footer-container"
>
  <h2>
    Math Magician project
  </h2>
  <h3>
    By, Tadesse Alemayehu.
  </h3>
</div>
`;
